webpack.dev.js

```diff
@@ -27,7 +27,8 @@ module.exports = {
                 options: {
                     presets: [
                         ['@babel/preset-env', {
-                            targets: "defaults"
+                            targets: "defaults",
+                            exclude: ["@babel/plugin-transform-regenerator"]
```

Comment:
This forces to use native async/await support in chrome which is better then babel transform. 
It will work onall browsers other then IE11.